### DISCUSSION TITLE
I have created a GitHub sponsor account and today I see that in my stripe connect account it is showing payment recieved and settled but I don't understand that in my github sponsor account the payout is showing as 0 . Even I have not received any payment in my local bank account

### DISCUSSION THREAD

**Discussion by hritikd3** [community/community#40928] [New to GitHub]
I am totally confused why it is not showing any payout received in my GitHub sponsor account but at the same time, it is showing in my stripe connect account that I have received a payout that got settled.

**Comment by queenofcorgis** [comment]
Hi there - thanks for writing in, for personal account matters, it is best to open a [support ticket](https://support.github.com/contact/account?type=github_sponsors) with our Sponsors team. I've let the team know about your post, but feel free to discuss more personal info with them at the link provided.
